CONTRIBUTING.md

```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
 ## LibreFoodPantry
 
 This project is a [LibreFoodPantry](https://librefoodpantry.org) project.
-We are a member of this larger community and shares its resources
-and policies including its
+We are a part of this larger community and shares its policies and resources.
 
-- Vision
-- Mission
-- Values
-- Code of Conduct
-- Licensing
-- Communication Channels
-- Governance
 
-[Please read them](https://librefoodpantry.org),
-uphold them, and use them when contributing to this project.
+## Values
 
+To ensure a healthy and safe environment in which to collaborate and learn,
+and to help establish and promote effective development practices,
+we have adopted the following values.
+We expect all community members to read and uphold these values.
+
+* [Code of conduct](CODE_OF_CONDUCT.md)
+* Agile [values](https://agilemanifesto.org/)
+  and [principles](https://agilemanifesto.org/principles.html)
+* [FOSSisms](https://opensource.com/education/14/6/16-foss-principles-for-educators)
 
-## Communication
 
-In general:
+## Licensing
 
-* Use the issue tracker to report problems,
-propose features, strategize, give feedback,
-propose changes to policies and procedures,
-offer changes through merge requests, review changes, etc.
-* Use chat to ask questions, schedule meetings, hold meetings,
-socialize, etc.
-* Use email for personal, private matters.
+We license source code under [GPL v3](LICENSE.md)
+and content under [GFDL v1.3](LICENSE_FOR_CONTENT.md).
+For each commit, contributors must
+* Include a [copyright notice](copyright-notices/) on each new file
+* Sign-off on the [DCO v1.1](DCO.txt) in the commit message
+* And credit all co-authors in the commit message
 
+Please read
+[Licensing](docs/dev/licensing.md)
```

Comment:
This doesn't link to anything (no document named "licensing" in this folder).

Reply:
It should be there; I renamed docs/dev/copyright-licensing-and-dco.md → docs/dev/licensing.md . I was able to test this link by viewing the "code" tab and changing the branch to "contributing", then opening CONTRIBUTING.md and clicking on the link.

I'm marking this one resolved.